python/xorbits/core/utils/docstring.py

```diff
@@ -54,6 +54,7 @@ def add_docstring_disclaimer(
     docstring_src_module: Optional[ModuleType],
     docstring_src_cls: Optional[Callable],
     doc: Optional[str],
+    fallback_warning: Optional[bool] = False,
```

Comment:
Seems `fallback_warning` will never be `None`, so its type should simply be `bool`.